foundry.toml

```diff
@@ -14,7 +14,7 @@ script = "contracts/scripts" # The path to the contract scripts relative to the
 solc_version = "0.8.24" # Uses a single Solidity compiler version for the project
 src = "contracts" # The path to the contract sources relative to the root of the project.
 test = "contracts/test" # The path to the test contract sources relative to the root of the project.
-cache_path = "contracts/cache" # The path to the cache, relative to the root of the project.
+cache_path = "contracts/contracts-cache" # The path to the cache, relative to the root of the project.
```

Comment:
turbo needs to be aware of this directory, but once include a directory in turbo cache, you include it on every package in the workspace. and we don't want the `cache` directory of every package in turbo cache, so i'm giving this one a more specific name